source/docs/b3.enums.md

```diff
@@ -20,7 +20,7 @@ enum Day {
 
 ⭐️ Variants can be accessed through :: notation , ex. Day::Sunday
 
-⭐️ Each enum **variant** can have,
+⭐️ Each enum **variant** can have:
```

Comment:
I tried to avoid using : because we are talking about :: notation in here

Reply:
Yes, but in this context, I don't see the problem using `:`. But if you really want to stick with it, I can remove the change.

Reply:
sorry buddy for the late response, prefer if you can remove